backend/src/main/java/gov/cdc/usds/simplereport/api/uploads/FileUploadController.java

```diff
@@ -0,0 +1,49 @@
+package gov.cdc.usds.simplereport.api.uploads;
+
+import static gov.cdc.usds.simplereport.config.WebConfiguration.PATIENT_UPLOAD;
+import static gov.cdc.usds.simplereport.config.WebConfiguration.RESULT_UPLOAD;
+
+import gov.cdc.usds.simplereport.api.model.errors.CsvProcessingException;
+import gov.cdc.usds.simplereport.db.model.TestResultUpload;
+import gov.cdc.usds.simplereport.service.TestResultUploadService;
+import gov.cdc.usds.simplereport.service.UploadService;
+import java.io.IOException;
+import java.io.InputStream;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+public class FileUploadController {
+  private final UploadService uploadService;
+  private final TestResultUploadService testResultUploadService;
+
+  @PostMapping(PATIENT_UPLOAD)
+  public String handlePatientsUpload(@RequestParam("file") MultipartFile file) {
```

Comment:
Is there a way to put some stricter typing/validation on these multipart files? Such that we can reject if it's not a CSV? In particular, I'm worried that since this won't be applying graphQL's argument limits anymore, theoretically people could try to upload massive or incorrectly formatted files - it's a bit of a security concern, even though this endpoint is only open to authenticated users.

Reply:
Looks like it's a Spring configuration we can enable: [spring.servlet.multipart.max-file-size](https://spring.io/guides/gs/uploading-files/#:~:text=spring.servlet.multipart.max%2Dfile%2Dsize%3D128KB)

Reply:
I am also adding an assert on the type of the file, even though that does nothing to the security since file extensions are a syntactical sugar on files

Reply:
I dont think i ever heard on `clamav`, it could be something we can look into, doesnt seem like a quick thing to integrate with spring though

Reply:
no I wasn't suggesting it for this PR 😄 maybe just a future thing if we want to bump up our security

Reply:
also from what I remember it was not too bad to integrate with java/spring but not a quick add